src/test/java/org/folio/rest/api/MaterialTypeTest.java

```diff
@@ -7,6 +7,8 @@
 import io.vertx.core.http.HttpClientResponse;
 import io.vertx.core.http.HttpMethod;
 import io.vertx.core.json.JsonObject;
+
+import org.folio.rest.jaxrs.model.Status;
```

Comment:
What is this import used for?

Reply:
Thanks, good catch. I wonder, why sonar did not notify about it...